docs/README.md

```diff
@@ -0,0 +1,80 @@
+# IBM Cloud Internet Service (CIS)
+IBM Cloud Internet Services, powered by Cloudflare, provides a fast, highly performant, reliable, and secure internet service for customers running their business on IBM Cloud. The CIS provides security, reliabilty and performance features to configure for your domain such as web application firewall (WAF), distributed denial of service (DDoS), global load balancing (GLB) and domain name system(DNS). For more information see, [About IBM Cloud Internet Services](https://cloud.ibm.com/docs/cis?topic=cis-about-ibm-cloud-internet-services-cis).
+
+## Domain
+
+The [domain module](https://github.com/terraform-ibm-modules/terraform-ibm-cis/blob/main/modules/domain/) provides the terraform resources required to add and configure a domain in a CIS instance. After the module run successfully-
+
+* The status of the domain that is configured in the CIS instance is set to `pending`.
+* You then configure the name servers that are assigned to the domain at the DNS provider or registrar.
+* You can find the assigned name servers in the `name_servers` variable of the module output.
+* The status changes to `active` after the name servers are configured correctly at the DNS provider or registrar.
+
+For more information, see [Domain lifecycle concepts](https://cloud.ibm.com/docs/cis?topic=cis-domain-lifecycle-concepts).
+
+## Global Load Balancer(GLB)
+
+The global load balancing service efficiently allocates your traffic among numerous servers by utilizing a blend of origin pools, health checks, and a load balancer. Global load balancing encompasses the following functionalities:
+- Load balancing options with both proxy and non-proxy configurations.
+- Utilization of origin pools and health checks.
+
+The [glb module](https://github.com/terraform-ibm-modules/terraform-ibm-cis/blob/main/modules/glb/) provides terraform resources to create and manage global load balancers in a CIS instance. It also allows to configure health checks and other settings. For more information see [global load balancer concepts](https://cloud.ibm.com/docs/cis?topic=cis-global-load-balancer-glb-concepts).
+
+* The module runs without any error but when `glb_proxied` is set as `true`, then `ttl` is automatically set and cannot be updated.
```

Comment:
I wont say that module gives error when glb_proxied is true.
So use the same line as it was.